x-pack/platform/plugins/shared/cases/common/types/domain/suggestion/v1.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { z } from '@kbn/zod';
+import type { CaseAttachmentWithoutOwner } from '../attachment/v1';
+import { OWNERS } from '../../../constants/owners';
+
+const suggestionOwnerSchema = z.enum(OWNERS);
+
+export const suggestionContextRt = z.object({
+  spaceId: z.string(),
+  'service.name': z.array(z.string()).optional(),
+  timeRange: z
+    .object({
+      from: z.string(),
+      to: z.string(),
+    })
+    .optional(),
+});
+
+export type SuggestionContext = z.infer<typeof suggestionContextRt>;
+export type SuggestionOwner = z.infer<typeof suggestionOwnerSchema>;
+export type SuggestionOwners = SuggestionOwner[];
+export type GenericSuggestionPayload = object;
```

Comment:
What do you for see being a part of the payload? It doesn't seem like it's being used atm?